pkg/distributor/distributor_test.go

```diff
@@ -994,6 +999,79 @@ func TestDistributor_QueryStream_ShouldReturnErrorIfMaxSeriesPerQueryLimitIsReac
 	_, err = ds[0].QueryStream(ctx, math.MinInt32, math.MaxInt32, allSeriesMatchers...)
 	require.Error(t, err)
 	assert.Contains(t, err.Error(), "max number of series limit")
+
+}
+
+func TestDistributor_QueryStream_ShouldReturnErrorIfMaxChunkBytesPerQueryLimitIsReached(t *testing.T) {
+	const seriesToAdd = 10
+	// This is used to track our initial test series to calculate response chunk size.
+	const initialSeries = 1
```

Comment:
You can remove this, and just add `seriesToAdd-1` below, after the 1st series has been pushed.